docs/virtual_machines/host-devices.md

```diff
@@ -77,16 +77,13 @@ configuration:
 
  * `pciVendorSelector` is a combination of a `vendor_id:product_id` required for a device identification on a host. This identifier `10de:1eb8` can be found using `lspci`.
 
-	```
-	lspci -nnv|grep -i nvidia
-	65:00.0 3D controller [0302]: NVIDIA Corporation TU104GL [Tesla T4] [10de:1eb8] (rev a1)
-	```
+        lspci -nnv|grep -i nvidia
+        65:00.0 3D controller [0302]: NVIDIA Corporation TU104GL [Tesla T4] [10de:1eb8] (rev a1)
```

Comment:
This is incorrect.  The correct diff should should be 

```console
    $ lspci -nnv|grep -i nvidia
    65:00.0 3D controller [0302]: NVIDIA Corporation TU104GL [Tesla T4] [10de:1eb8] (rev a1)
```

Reply:
Sure, I can add the missing `$`.